.buildkite/pipeline.trigger.compliance.tests.sh

```diff
@@ -28,6 +28,7 @@ steps:
 EOF
 
 # Generate each test we want to do.
+compliance_test 9.1.0-SNAPSHOT 3.4.0
```

Comment:
This would allow to test the installation of good_content package that contains SLO assets.

```
  @3.4.0
  Scenario: Content package can be installed
   Given the "good_content" package is installed
   #Then there are no errors.
```

Does it make sense?

Reply:
Or maybe, just use the new `basic_content` and remove the testing of `good_content` (this contains SLO assets)?

Reply:
Compliance tests for 3.4.0 in 9.1.0 should fail, because 3.4.0 is not implemented yet. Or is it?

Reply:
It's not failing the current compliance tests, but I don't see any SLO assets created when I tested locally to install in Elastic stack 9.1.0-SNAPSHOT (related PR not merged yet https://github.com/elastic/kibana/pull/186974).

Reply:
It has been updated the compliance test using the `good_content` package to validate that there is a specific SLO. Currently, it's failing when using spec 3.4.0 and latest Elastic stack 9.1.0-SNAPSHOT.

https://buildkite.com/elastic/package-spec/builds/1053#0195c88d-3a38-44db-a668-f1c14f0cf477

Therefore, I will remove the compliance test for that scenario (9.1.0-SNAPSHOT and spec 3.4.0): https://github.com/elastic/package-spec/pull/884/commits/5b59414583a7ead4e5fea4145b3760bb3578463c